torch/csrc/jit/codegen/cuda/test/test_gpu_view.cpp

```diff
@@ -2168,7 +2168,7 @@ TEST_F(NVFuserTest, FusionIssue2076_CUDA) {
   auto t12 = t4 + t11;
   auto t13 = t12.view({48, 128, 128});
   // 48, 128, 128
-  auto t14 = std::get<0>(t13.max({2}));
+  auto t14 = std::get<0>(t13.max(2));
```

Comment:
wasn't this a compile time error before the fix?!

Reply:
It's not an error, but the parameter of `max` is just a scalar value, so clang was giving a warning that `{}` shouldn't be used.

Reply:
```
../torch/csrc/jit/codegen/cuda/test/test_gpu_view.cpp:2171:34: warning: braces around scalar initializer [-Wbraced-scalar-init]
  auto t14 = std::get<0>(t13.max({2}));
```